src/commands/runDraftTool/runDraftUpdate.ts

```diff
@@ -88,6 +166,7 @@ async function multiStepInput(context: ExtensionContext, destination: string) {
 
     if (isSuccess) {
 	    window.showInformationMessage("Web app routing annotation succeeded");
+		window.showInformationMessage("Deploy using the draft generated files");
```

Comment:
This part will be a button on the other information message in the future but fine for now. Good stuff!

Example from another scenario
```typescript
	    window.showInformationMessage(`Draft Dockerfile Succeeded`, buildContainer)
            .then(option => {
                if (option === buildContainer) { // do something here }
            });
```